nixos/modules/config/pulseaudio.nix

```diff
@@ -254,24 +254,35 @@ in {
       };
     })
 
-    (mkIf systemWide {
+    (mkIf systemWide (let
+      cookiePath = "${stateDir}/.config/pulse/cookie";
```

Comment:
Put this line below `stateDir =`, there is no need for the extra `let`.

Reply:
cookiePath is only needed in case of systemWide.

Reply:
`stateDir` too.